script/test/cmd/tests_new.sh

```diff
@@ -82,3 +82,16 @@ convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/script/test/fixture
 #TEST the pvc-request-size command parameter
 convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/script/test/fixtures/pvc-request-size/docker-compose.yml convert -o $TEMP_DIR/output_dir2/output-k8s.json -j --pvc-request-size=300Mi" "$TEMP_DIR/output_dir2/output-k8s.json"
 convert::check_artifacts_generated "kompose --provider=openshift -f $KOMPOSE_ROOT/script/test/fixtures/pvc-request-size/docker-compose.yml convert -o $TEMP_DIR/output_dir2/output-os.json -j --pvc-request-size=300Mi" "$TEMP_DIR/output_dir2/output-os.json"
+
+######
+# Test the path of build image
+# Test build v2 absolute compose file
+convert::check_artifacts_generated "kompose --build local -f $KOMPOSE_ROOT/script/test_in_openshift/compose-files/buildconfig/docker-compose.yml convert -o $TEMP_DIR/output_file" "$TEMP_DIR/output_file"
```

Comment:
Most of the testdir has been deprecated, try using `script/fixtures/buildconfig/` as your fixtures.

Reply:
Ok, I have fixed it.